dataprofiler/profilers/utils.py

```diff
@@ -462,26 +491,18 @@ def find_diff_of_dates(stat1, stat2):
     :type stat1: datetime.datetime object
     :param stat2: the second statistical input
     :type stat2: datetime.datetime object
-    :return: Difference in stats
-    :rtype: str
+    :return: difference in stats
+    :rtype: Union[List, str]
     """
-    # We can use find_diff_of_numbers since datetime objects
-    # can be compared and subtracted naturally
-    diff = find_diff_of_numbers(stat1, stat2)
-    if isinstance(diff, str):
-        return diff
-    if isinstance(diff, list):
-        # Deal with the case where one stat is None
-        return [None if i is None else i.strftime("%x %X") for i in diff]
-
-    # Must be timedelta object
-    if diff.days >= 0:
-        return "+" + str(diff)
-
-    return "-" + str(abs(diff))
+    if stat1 is None or stat2 is None:
```

Comment:
I believe this will actually error bc we have an error in our test.
Line 107 in : https://github.com/capitalone/DataProfiler/blob/69d71736ca032b20a290c5573f9b3c7a9d2b1577/dataprofiler/tests/test_utils.py
```python
self.assertEqual("unchanged", utils.find_diff_of_numbers(None, None))
```
should be:
```python
self.assertEqual("unchanged", utils.find_diff_of_dates(None, None))
```

As a result, this passes currently, but would with the proper test.

Can we add this fix to the test in the PR